x-pack/legacy/plugins/canvas/public/components/workpad_header/workpad_zoom/workpad_zoom.tsx

```diff
@@ -63,22 +95,35 @@ export class WorkpadZoom extends PureComponent<Props> {
     }));
 
   _getPanels = (): EuiContextMenuPanelDescriptor[] => {
-    const { zoomScale, zoomIn, zoomOut } = this.props;
+    const { zoomScale, zoomIn, zoomOut, resetZoom } = this.props;
     const items: EuiContextMenuPanelItemDescriptor[] = [
+      {
+        name: 'Fit to window',
+        icon: 'empty',
+        onClick: this._fitToWindow,
+        disabled: zoomScale === MAX_ZOOM_LEVEL,
+      },
       ...this._getScaleMenuItems(),
       {
         name: 'Zoom in',
-        icon: 'starPlusFilled', // TODO: change this to magnifyWithPlus when available
+        icon: 'magnifyWithPlus',
         onClick: zoomIn,
         disabled: zoomScale === MAX_ZOOM_LEVEL,
         className: 'canvasContextMenu--topBorder',
       },
       {
         name: 'Zoom out',
-        icon: 'starMinusFilled', // TODO: change this to magnifyWithMinus when available
+        icon: 'magnifyWithMinus',
         onClick: zoomOut,
         disabled: zoomScale === MIN_ZOOM_LEVEL,
       },
+      {
+        name: 'Reset',
+        icon: 'empty',
+        onClick: resetZoom,
+        disabled: zoomScale === MAX_ZOOM_LEVEL,
+        className: 'canvasContextMenu--topBorder',
```

Comment:
`--` or `__`?

Reply:
Yep `--` because it's a modifier, and we use the BEM naming convention for classes. http://getbem.com/naming/. This class just adds a top border to the element, but it's not used to identify a sub-element or separate component. It would be `__` if it was like `canvasContextMenu__menuItem`.